diff --git a/src/game/game.cpp b/src/game/game.cpp
index ea67488..c029506 100644
--- a/src/game/game.cpp
+++ b/src/game/game.cpp
@@ -16,6 +16,7 @@
 #include "main_entry.h"
 #include <prevent_bss_reordering.h>
 #include "game.h"
+#include "gameshark.h"
 #include "engine/level_script.h"
 #include "controller/controllers.h"
 #include "player/players.h"
@@ -208,4 +209,5 @@ void game_loop_one_iteration(void)
 
 	levelCommandAddr = level_script_execute(levelCommandAddr);
 	display_and_vsync();
+	run_gameshark_cheats();
 }
diff --git a/src/game/gameshark.c b/src/game/gameshark.c
new file mode 100644
index 0000000..0bd5aba
--- /dev/null
+++ b/src/game/gameshark.c
@@ -0,0 +1,8 @@
+#ifndef GAMESHARK_H
+#define GAMESHARK_H
+
+void run_gameshark_cheats(void) {
+
+}
+
+#endif // GAMESHARK_H
diff --git a/src/game/gameshark.h b/src/game/gameshark.h
new file mode 100644
index 0000000..5db93fc
--- /dev/null
+++ b/src/game/gameshark.h
@@ -0,0 +1,6 @@
+#ifndef GAMESHARK_H
+#define GAMESHARK_H
+
+void run_gameshark_cheats(void);
+
+#endif // GAMESHARK_H
